example/pubspec.yaml

```diff
@@ -15,6 +15,7 @@ dependencies:
   json_dynamic_widget:
     path: ../
   websafe_svg: ^1.1.1
+  json_theme: ^1.2.1
```

Comment:
Is this really needed?  This should be being picked up via the library import and defining it here can create conflicts between the example and the library the example is using.